static/app/views/issueList/hotspots.tsx

```diff
@@ -0,0 +1,276 @@
+import React, {useEffect, useState} from 'react';
+import * as d3 from 'd3-hierarchy';
+import {CustomChart} from 'echarts/charts';
+import {
+  DatasetComponent,
+  GridComponent,
+  TitleComponent,
+  TooltipComponent,
+  VisualMapComponent,
+} from 'echarts/components';
+import * as echarts from 'echarts/core';
+import {SVGRenderer} from 'echarts/renderers';
+import ReactEChartsCore from 'echarts-for-react/lib/core';
+
+import * as Layout from 'sentry/components/layouts/thirds';
+import type {Project} from 'sentry/types';
+
+// Register the required components
+echarts.use([
+  TitleComponent,
+  TooltipComponent,
+  DatasetComponent,
+  GridComponent,
+  VisualMapComponent,
+  CustomChart,
+  SVGRenderer,
+]);
+
+type Props = {
+  organizationSlug: string;
+  projects: Project[];
+};
+
+function IssueHotSpots({organizationSlug, projects}: Props) {
+  const DELIMITER = '/';
+
+  const [isLoading, setIsLoading] = useState(true);
+
+  const [diagramData, setDiagramData] = useState();
+
+  const projectId = projects[0].id;
+
+  useEffect(
```

Comment:
You should use `AsyncComponent` to handle data fetching instead of building this yourself